app/views/ViewDates.scala

```diff
@@ -16,18 +16,23 @@
 
 package views
 
-import java.time.{ZoneId, ZoneOffset}
+import java.time.ZoneId
 import java.time.format.DateTimeFormatter
 import java.time.temporal.TemporalAccessor
 
 object ViewDates {
 
-  val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
-  val submissionDateTimeFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM uuu 'at' HH:mm").withZone(ZoneId.of("Europe/London"))
-  val dateFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM uuu").withZone(ZoneId.of("Europe/London"))
+  private val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
```

Comment:
Made the formatters private to force consumers to use the "format" methods.
BTW - I have a feeling that the "yyyy-MM-dd HH:mm" above isn't used anywhere.  I'll investigate and create a new PR if that is the case.